### DISCUSSION TITLE
Happy New Year!

### DISCUSSION THREAD

**Discussion by Meme-Eater** [Monika-After-Story/MonikaModDev#8882] [appreciation]
I just want to say Thank you guys, and wish you a very Happy New Year! You guys have done such an amazing work, I can't really find words to express my gratitude.

And the dress. Man...that dress....My Monika looks gorgeous!

I hope you guys had a happy 2019, and I wish you a great 2020!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
